### PR TITLE
refactor: extra attributes for details-box-mobile component

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

Related to https://app.clickup.com/t/1e6u51q

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: resources/views/details-box-mobile.blade.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="flex justify-between">
     <div class="{{ $titleClass ?? 'w-32' }}">{{ $title }}</div>
 
-    <div class="flex items-center space-x-3">
-        <div>{{ $slot }}</div>
+    <div class="flex items-center space-x-3 {{ $wrapperClass ?? false }}">
+        <div class="{{ $slotClass ?? 'false' }}">{{ $slot }}</div>
```
Comment (ItsANameToo) [inline resources/views/details-box-mobile.blade.php:5]: won't this result in a `class="false"` entry?

Comment (ItsANameToo) [inline resources/views/details-box-mobile.blade.php:5]: is this needed or could it be passed a `div` with the proper classes as part of the slot instead?

File: resources/views/details-box-mobile.blade.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <div class="flex justify-between">
     <div class="{{ $titleClass ?? 'w-32' }}">{{ $title }}</div>
 
-    <div class="flex items-center space-x-3">
-        <div>{{ $slot }}</div>
+    <div class="flex items-center space-x-3 {{ $wrapperClass ?? false }}">
```
Comment (ItsANameToo) [inline resources/views/details-box-mobile.blade.php:4]: can these be changed to properties?

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/details-box-mobile.blade.php (hunk 1 above)
Comment (Highjhacker) [reply resources/views/details-box-mobile.blade.php:5]: Oh my bad, that slipped.

Comment (Highjhacker) [reply resources/views/details-box-mobile.blade.php:5]: Was needed for my usecase, need to apply on both the slot & the parent div

Comment (Highjhacker) [reply resources/views/details-box-mobile.blade.php]: https://github.com/ArkEcosystem/nodem/pull/245/files#diff-6d86f9014b05fcb82fee7fcb48e7770d0343e323886a08c71de75f418477589dR53

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**